src/mlte/measurement/memory/local_process_memory_consumption.py

```diff
@@ -239,3 +239,5 @@ def _get_memory_usage(pid: int) -> int:
         return int(used.decode("utf-8").strip()[:-1])
     except ValueError:
         return 0
+    except FileNotFoundError as e:
+        raise RuntimeError("External program needed to get memory usage was not found: " + str(e))
```

Comment:
Same as previous comment. Refactor to f-string.

Reply:
Ok